changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 4.5.12 (2020-07-01)
+  Fixed so links with xlink:href attributes are filtered correctly to prevent XSS. #TINY-1626
```

Comment:
"Fixed so..." feels like odd wording, but being a backport I guess that's what we used before? Oh well.

Reply:
Yeah, I just used the original changelog verbatim. We can probably change it.